assets/models/system/stabilityai-stable-diffusion-xl-base-1.0/spec.yaml

```diff
@@ -0,0 +1,29 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: stabilityai-stable-diffusion-xl-base-1.0
```

Comment:
```suggestion
name: stabilityai-stable-diffusion-xl-base-1-0
```

Reply:
Let's follow same naming convention across the board..
For example, we have "Salesforce-BLIP-2-opt-2-7b-image-to-text"

Reply:
Changed the Directory name Thanks